<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterStudent</class>
 <widget class="QDialog" name="RegisterStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterStudent</string>
  </property>
  <widget class="QGroupBox" name="RegiStudForm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>631</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aldhabi</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#RegiStudForm{
background :rgb(170, 255, 255)
}</string>
   </property>
   <property name="title">
    <string>Formulaire pour les Clients</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>45</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>148</y>
      <width>64</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Prenom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PrenomEtud">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>219</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>252</y>
      <width>83</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameStudRegi">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>322</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordEtud">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>355</y>
      <width>76</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEtudRegi">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>426</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>459</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Numero de Telephone </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SpecEtud">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="studButtomRegister">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registrer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NomEtud">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>115</y>
      <width>421</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="LoginEtud">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>540</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>studButtomRegister</tabstop>
  <tabstop>PrenomEtud</tabstop>
  <tabstop>UsernameStudRegi</tabstop>
  <tabstop>PasswordEtudRegi</tabstop>
  <tabstop>SpecEtud</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
